chain-impl-mockchain/src/tokens.rs

```diff
@@ -0,0 +1,208 @@
+use chain_core::mempack::{ReadBuf, ReadError, Readable};
+use cryptoxide::{blake2b::Blake2b, digest::Digest};
+use thiserror::Error;
+use typed_bytes::ByteBuilder;
+
+use std::convert::TryInto;
+
+pub const POLICY_HASH_SIZE: usize = 28;
+pub const TOKEN_NAME_MAX_BYTES: usize = 32;
```

Comment:
nit: would be nice to have the same convention for names, e.g. both having `BYTES`/`SIZE` as suffix